Helper/ApiClientHelper.php

```diff
@@ -59,7 +61,13 @@ public function getJsinit()
         $settings = [
             "enabled" => $this->getStoreConfig('postcodenl_api/general/enabled'),
             "supported_countries" => json_encode($this->formatSupportedCountriesJs($this->getStoreConfig('postcodenl_api/general/supported_countries'))),
-            "debug" => $this->isDebugging()
+            "nl_input_behavior" => $this->getStoreConfig('postcodenl_api/general/nl_input_behavior'),
+            "debug" => $this->isDebugging(),
+            "translations" => [
+                'flekto_nl_zip_label' => __('Postcode and house number'),
+                'flekto_nl_zip_placeholder' => __('1234AB 1'),
```

Comment:
Please don't use 1234AB: https://www.vice.com/nl/article/wj8b4x/wonen-op-postcode-1234-ab-is-een-bureaucratische-hel-vol-neppost

Reply:
I don't think it's applicable in this case.

It's a placeholder for an example, not a pre-filled in value. User should still fill in the field.
Fake mail is something different than package delivery.
Plugin users can change it in lang files if they need something other than 1234AB

Reply:
FYI: 1234AB does not actually exist anyway: https://www.postcode.nl/zoek/1234ab